api/src/main/resources/workbench-api.yaml

```diff
@@ -5091,6 +5122,15 @@ definitions:
       jwt:
         type: string
         description: the encoded/signed JWT
+  RasLinkRequestBody:
+    required:
+    - authcode
+    type: object
+    properties:
+      authcode:
```

Comment:
Is this specific casing used by the oauth2 spec? If so - this is good for consistency, otherwise if you were writing this from scratch I would probably go `authCode`